waspc/src/Wasp/Util/IO.hs

```diff
@@ -15,6 +18,8 @@ import qualified System.Directory as SD
 import qualified System.FilePath as FilePath
 import System.IO.Error (isDoesNotExistError)
 import UnliftIO.Exception (catch, throwIO)
+import Prelude hiding (readFile)
+import qualified Prelude as P
```

Comment:
`readFile` is imported by default. This is how we hide it so we can define our own (the same applies to files that use our implementation instead of the default one).